conda/notices/cache.py

```diff
@@ -81,14 +87,24 @@ def get_notices_cache_dir() -> Path:
 
 
 def get_notices_cache_file() -> Path:
-    """Returns the location of the notices cache file as a Path object"""
+    """
+    Returns the location of the notices cache file as a Path object
+
+    If the file does not exist, we create it and set the "created_at" time stamp
+    of the file so that it is in the past. This forces the notices to be checked
+    for and read later.
+    """
     cache_dir = get_notices_cache_dir()
     cache_file = cache_dir.joinpath(NOTICES_CACHE_FN)
 
     if not cache_file.is_file():
         with open(cache_file, "w") as fp:
             fp.write("")
 
+        current_time = time.time()
+        create_time = current_time - NOTICES_DECORATOR_DISPLAY_INTERVAL
+        os.utime(cache_file, (create_time, create_time))
```

Comment:
This seems really odd. Can we write the timestamp to a file that has true filesystem timestamps (i.e. the current time?) Recommend using mtime only as the timestamp conda cares about.